src/WebJobs.Script/Workers/ProcessManagement/DefaultWorkerProcessFactory.cs

```diff
@@ -89,5 +100,21 @@ internal string SanitizeExpandedArgument(string envExpandedString)
             }
             return envExpandedString;
         }
+
+        internal void ApplyWorkerConcurrencyLimits(ProcessStartInfo startInfo)
+        {
+            // Set higher concurrency limits for Python and Powershell language workers for V4
```

Comment:
You mention this is only for v4 here, but this PR targets dev?

Reply:
it was copy pasted from: https://github.com/Azure/azure-functions-host/commit/d45892af1d049d066edb92b449fe131ffc062776
removed the comment